### ISSUE TITLE
Problem: Levels of description model specifies max_length as a string and not an integer

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#1243]
**Expected behaviour**

When migrations are run in Archivematica they run cleanly. 

**Current behaviour**

We noticed the following at a client site:
```
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

We were able to repeat this issue in Docker, and Bionic. After investigation it was clear this was due to the `Levels of description` model. 

**Steps to reproduce**

_Docker_

* `sudo make manage-dashboard ARG=migrate main`

_Bionic_

```
sudo -u archivematica bash -c " \
    set -a -e -x
    source /etc/default/archivematica-dashboard || \
        source /etc/sysconfig/archivematica-dashboard \
            || (echo 'Environment file not found'; exit 1)
    cd /usr/share/archivematica/dashboard
    /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python manage.py migrate --noinput
";
```

_Docker_

I ran the following command to understand the issue: 

* `sudo make manage-dashboard ARG="makemigrations --dry-run --verbosity 3"` 

_Bionic_

```
artefactual@am111bionic:~$ sudo -u archivematica bash -c " \
    set -a -e -x
    source /etc/default/archivematica-dashboard || \
        source /etc/sysconfig/archivematica-dashboard \
            || (echo 'Environment file not found'; exit 1)
    cd /usr/share/archivematica/dashboard
    /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python manage.py makemigrations --dry-run --verbosity 3
```

_Result_

```
Migrations for 'main':
  0076_auto_20200629_1520.py:
    - Alter field name on levelofdescription
Full migrations file '0076_auto_20200629_1520.py':
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0075_userprofile_system_emails'),
    ]

    operations = [
        migrations.AlterField(
            model_name='levelofdescription',
            name='name',
            field=models.CharField(max_length=b'1024'),
        ),
    ]
```
It is still a little abstract, but `(max_length=b'1024')` is a decent clue something might be amiss because a length field should not have a byte encoding. 

Cross-checking this with the code I can see [this line](https://github.com/artefactual/archivematica/blob/396015a44723e6b2dd2f4d7bed1443514a429c12/src/dashboard/src/main/models.py#L1609) repeats the pattern and encodes max_length as a string. 

Once this is turned back into an integer, the models/database align correctly. (I'm going to guess this is mysql being more clever than Django and dealing in integers regardless :shrug: but who knows.

```
ross-spencer@artefactual:~/git/artefactual-labs/am/compose$ sudo make manage-dashboard ARG="makemigrations --dry-run --verbosity 3"
docker-compose run \
	--rm \
	--entrypoint /src/dashboard/src/manage.py \
		archivematica-dashboard \
			makemigrations --dry-run --verbosity 3
Starting compose_mysql_1 ... done
Starting compose_redis_1         ... done
Starting compose_archivematica-storage-service_1 ... done
Starting compose_gearmand_1                      ... done
No changes detected
```

**Your environment (version of Archivematica, operating system, other relevant details)**

As above. 

**tl;dr**

This should clear things up. 

```diff
diff --git a/src/dashboard/src/main/models.py b/src/dashboard/src/main/models.py
index 17099b1a..419610d9 100644
--- a/src/dashboard/src/main/models.py
+++ b/src/dashboard/src/main/models.py
@@ -1606,7 +1606,7 @@ class FileID(models.Model):
 
 class LevelOfDescription(models.Model):
     id = UUIDPkField()
-    name = models.CharField(max_length="1024")  # seems long, but AtoM allows this much
+    name = models.CharField(max_length=1024)  # seems long, but AtoM allows this much
     # sortorder should be unique, but is not defined so here to enable swapping
     sortorder = models.IntegerField(default=0, db_column="sortOrder")
```

**More information**

I investigated this to ensure that between, `1.10` and `1.11` we had introduced an error somewhere around Python 3 and field encoding. 

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by replaceafill** [comment]
This issue is being closed as part of a bulk closure process. Please click [here](https://groups.google.com/u/1/g/archivematica/c/Sq7uw1zloTA) for further information.
